packages/api/src/controllers/sensorController.js

```diff
@@ -565,7 +565,7 @@ const sensorController = {
   async retireSensor(req, res) {
     const trx = await transaction.start(Model.knex());
     try {
-      const { external_id, location_id, farm_id, partner_id, brand_name } = req.body.sensorInfo;
+      const { external_id, location_id, farm_id, partner_id, brand_name } = req.body;
```

Comment:
Client should have to send this data in the request body. Just the `location_id` should be enough to unclaim the sensor.

Reply:
You can pull the other data from the database.